packages/app-store/_utils/CRMRoundRobinSkip.ts

```diff
@@ -7,20 +8,13 @@ import { EventTypeMetaDataSchema } from "@calcom/prisma/zod-utils";
 
 export async function getCRMContactOwnerForRRLeadSkip(
   bookerEmail: string,
-  eventTypeId: number
+  eventTypeMetadata: Prisma.JsonValue
 ): Promise<string | undefined> {
-  const eventTypeMetadataQuery = await prisma.eventType.findUnique({
```

Comment:
Removed this extra roundtrip to the DB to get the metadata since we already had it.